scripts/qa_crowbarsetup.sh

```diff
@@ -3677,7 +3679,9 @@ function oncontroller_testsetup
         openstack security group rule create --protocol tcp --dst-port 1:65535 testvm
         openstack security group rule create --protocol udp --dst-port 1:65535 testvm
     else
-        nova secgroup-delete testvm || :
+        if nova secgroup-list-rules testvm &>/dev/null ; then
```

Comment:
Sorry for being so nitpicky today :smile: but why throw out both `stdout` and `stderr` here when you only `/dev/null` `stderr` above? Personally I prefer to only throw out `stderr` because `stdout` adds some more debug information which might be useful.

Reply:
reason I proposed dropping both is that we are just interested in the return code here to see 'does it exist already' - both the error message in case of non-existence and the information message on stdout in the other case would clutter up the log.

Reply:
That was my thinking too with the others, but like @bmwiedemann said, now we're listing the security group rules to test the existence of the group, and the rules themselves are kind of irrelevant.

Reply:
I don't know, I don't mind the extra output. But I am fine with it like this too.